### DISCUSSION TITLE
Remove limit on zoom out and ability to zoom out to show all blocks in single screen.

### DISCUSSION THREAD

**Discussion by ironmanoyo** [postmanlabs/postman-flows#91] [Ideas]
I had created flows with 25 blocks and will be adding more blocks to it, I had faced many issues with respect to zoom.
I was unable to view all blocks at once. (I was able to do in the web version but zooming out the page to 67%, but it was an hacky solution and I needed a solution in desktop client) and navigating huge flows are slow and a little bit buggier. 

To improve these things, I hope you guys can add the below features

1.  Remove limit on the zoom out
2. An button click will make an auto zoom out to an extent where all blocks are visible
3. Ability to change zoom sensitivity for trackpad/mouse. (It's too slow)

BTW, The product is great! Keep it up!

**Comment by samay-v** [comment]
Thanks for the feedback @ironmanoyo, We'll look into extending the zoom limits further. 

I am curious to see the flow that you created though, Would you be able to post a picture of it here?

**Reply by ironmanoyo** [reply]
Here you go

<img width="1510" alt="Screenshot 2021-10-29 at 3 11 04 PM" src="https://user-images.githubusercontent.com/45940611/139416314-d6ed8a02-7d4e-45c8-bfe5-571059c767b5.png">

Also, please take a look at my other ideas also

1. https://github.com/postmanlabs/postman-flows/discussions/84
2. https://github.com/postmanlabs/postman-flows/discussions/94
